<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />



<meta name="date" content="2024-06-24" />

<title>Summary: Mapping information-rich genotype-phenotype landscapes with genome-scale Perturb-seq</title>

<script src="site_libs/header-attrs-2.27/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.13.2/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-6.4.2/css/all.min.css" rel="stylesheet" />
<link href="site_libs/font-awesome-6.4.2/css/v4-shims.min.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>









<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">myproject</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/jliucx/myproject">
    <span class="fab fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Summary: Mapping information-rich
genotype-phenotype landscapes with genome-scale Perturb-seq</h1>
<h4 class="date">2024-06-24</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2024-07-09
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 2
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>myproject/</code> <span
class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.1). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date
</a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git
repository, you know the exact version of the code that produced these
results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomjliucxmyprojecttree78329171c8fef44d2e3f89c15504457ffe50c996targetblank7832917a">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong>
<a href="https://github.com/jliucx/myproject/tree/78329171c8fef44d2e3f89c15504457ffe50c996" target="_blank">7832917</a>
</a>
</p>
</div>
<div
id="strongRepositoryversionstrongahrefhttpsgithubcomjliucxmyprojecttree78329171c8fef44d2e3f89c15504457ffe50c996targetblank7832917a"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version
<a href="https://github.com/jliucx/myproject/tree/78329171c8fef44d2e3f89c15504457ffe50c996" target="_blank">7832917</a>.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/

Untracked files:
    Untracked:  analysis/images/

Unstaged changes:
    Modified:   README.md

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were
made to the R Markdown (<code>analysis/Analysis2.Rmd</code>) and HTML
(<code>docs/Analysis2.html</code>) files. If you’ve configured a remote
Git repository (see <code>?wflow_git_remote</code>), click on the
hyperlinks in the table below to view the files as they were in that
past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/jliucx/myproject/blob/78329171c8fef44d2e3f89c15504457ffe50c996/analysis/Analysis2.Rmd" target="_blank">7832917</a>
</td>
<td>
jliucx
</td>
<td>
2024-07-09
</td>
<td>
Publish second summary
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/jliucx/myproject/011bcf5eb4ae7bea4565de303d7e9cb1fbb3d971/docs/Analysis2.html" target="_blank">011bcf5</a>
</td>
<td>
jliucx
</td>
<td>
2024-07-07
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/jliucx/myproject/blob/08dd4b6e890b7a29e61bf92a4486491e11847923/analysis/Analysis2.Rmd" target="_blank">08dd4b6</a>
</td>
<td>
jliucx
</td>
<td>
2024-07-07
</td>
<td>
Add the second summary
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<div id="terminologies" class="section level2">
<h2>Terminologies</h2>
<ol style="list-style-type: decimal">
<li><strong>Phenotype-centric mapping</strong>: Approach that reveals
the genetic changes that drive a phenotype of interest.
<strong>Limitations</strong>: Pooled forward genetic screens typically
use low-dimensional phenotypes such as growth or marker expression for
selection. The use of simple phenotypes can conflate genes acting via
different mechanisms, requiring extensive follow-up studies to
disentangle genetic pathways. Additionally, in forward genetics,
serendipitous discovery is constrained by the prerequisite of selecting
phenotypes prior to screening.</li>
<li><strong>Gene-centric mapping</strong>: Catalogs the diverse
phenotypes caused by a defined genetic change.
<strong>Limitations</strong>: Typically been restricted in scale to
rationally chosen targets, limiting systematic comparisons.</li>
<li><strong>Incomplete penetrance</strong>: in the context of using the
Anderson-Darling test to detect transcriptional changes, it is referring
to the fact that not all cells with a genetic perturbation will show the
expected changes in gene expression. Some cells might carry the mutation
but not exhibit the altered transcriptional phenotype, leading to a
mixture of cells with and without the phenotype within the same
population. The AD test is useful in this context because it is
sensitive enough to detect changes in subsets of cells, thereby
identifying differentially expressed genes (DEGs) even when the
phenotype is not uniformly penetrant across all cells.</li>
</ol>
</div>
<div id="rationale" class="section level2">
<h2>Rationale</h2>
<div id="genome-scale-single-cell-crispr-screens"
class="section level3">
<h3>Genome-scale single-cell CRISPR Screens</h3>
<p>In genome-scale single-cell CRISPR screens, the diversity of
perturbations allows exploration of the range of cell states that can be
revealed by rich phenotypes. Additionally, as many human genes are well
characterized, these genes serve as natural controls to anchor
interpretation of comprehensive datasets. It explains what fraction of
genetic changes elicit transcriptional phenotypes and how
transcriptional responses differ between cell types.</p>
<p>Perturb-seq uses scRNA-seq to concurrently read out the CRISPR
single-guide RNAs (sgRNAs) (i.e., genetic perturbation) and
transcriptome (i.e., high-dimensional phenotype) of single cells in a
pooled format.</p>
</div>
<div id="multiplexed-crispri-strategy" class="section level3">
<h3>Multiplexed CRISPRi strategy</h3>
<ul>
<li>Compared with gain-of-function perturbations, a higher proportion of
loss-of-function perturbations yield phenotypes in growth and
chemical-genetic screens, especially for members of protein
complexes.</li>
<li>CRISPRi allows direct measurement of the efficacy of genetic
perturbation—knockdown—by scRNA-seq. Exploiting this feature allowed us
to target each gene in our library with a single element and empirically
exclude unperturbed genes from downstream analysis.</li>
<li>CRISPRi tends to yield more homogeneous perturbation than CRISPR
knockout, which can generate active in-frame indels. The relative
homogeneity of CRISPRi reduces selection for unperturbed cells,
especially when studying essential genes.</li>
<li>Unlike CRISPR knockout, CRISPRi does not lead to activation of the
DNA damage response which can alter transcriptional signatures.</li>
<li>To maximize CRISPRi efficacy, we used multiplexed CRISPRi libraries
in which each element contains two distinct sgRNAs targeting the same
gene. <strong>To avoid low representation of sgRNAs targeting essential
genes, we performed growth screens and, during oligonucleotide library
synthesis, overrepresented constructs that caused strong growth defects,
ensuring that critical sgRNAs are adequately represented, and mitigating
biases in the screening results.</strong></li>
</ul>
</div>
<div id="three-pronged-perturb-seq-screening-approach"
class="section level3">
<h3>Three-Pronged Perturb-seq Screening Approach</h3>
<ul>
<li><p>Using multiple time points allows for the observation of dynamic
changes and temporal effects of gene perturbation. Different cell types
can provide insights into cell-specific responses and identify universal
versus context-specific gene functions.</p></li>
<li><p>CML K562 Cells are a well-established model for chronic myeloid
leukemia, making them suitable for studying cancer-related gene
functions. RPE1 cells are non-cancerous, hTERT-immortalized, and exhibit
near-normal chromosomal content, making them a good model for studying
non-cancerous cell biology.</p></li>
<li><p>Screening Methodology: 10x Genomics Droplet-Based 3’ scRNA-seq
and Direct sgRNA Capture</p></li>
<li><p>Exclusion of Doublets: Cells with multiple sgRNAs were excluded
to avoid confounding effects from multiple gene perturbations. High
Coverage: Achieving a median coverage of over 100 cells per perturbation
ensures robust statistical power and reliable detection of gene
expression changes.</p></li>
</ul>
</div>
<div
id="a-robust-computational-framework-to-detect-transcriptional-phenotypes"
class="section level3">
<h3>A robust computational framework to detect transcriptional
phenotypes</h3>
<ul>
<li><p>Include many control cells bearing diverse non-targeting sgRNAs.
This allows internal z-normalization of measurements to correct for
batch effects.</p></li>
<li><p>Energy Distance Test: A global transcriptional changes test was
performed using a permuted energy distance test, comparing cells with
genetic perturbations to control cells. Approximately 31.1% of 9,608
perturbations targeting a primary transcript caused significant
transcriptional phenotypes in K562 cells. (<strong>Global
test</strong>)</p></li>
<li><p>Anderson-Darling (AD) Test: Used to detect differentially
expressed genes (DEGs) with incomplete penetrance, the AD test found
30.5% of perturbations caused &gt;10 DEGs in K562 cells.
(<strong>Multiple test</strong>)</p></li>
<li><p>Features of Genetic Perturbations: The strength of the
transcriptional response correlated with the growth phenotype, and
essential perturbations leading to significant transcriptional responses
were identified. Notably, many perturbations caused transcriptional
changes without affecting growth, indicating an influence on cell
state.</p></li>
</ul>
</div>
<div id="annotating-gene-function-from-transcriptional-phenotypes"
class="section level3">
<h3>Annotating gene function from transcriptional phenotypes</h3>
<ul>
<li><strong>Core idea: For different genes, if their perturbations have
similar transcriptional phenotypes, then their function might be
similar. Hence cluster perurbations with similar transcriptional
phenotypes.</strong></li>
<li>Use correlation between mean expression profile of gene
perturbations as a scale-invariant metric of similarity.</li>
<li>Use CORUM3.0 and STRING databese to validate that perturbations with
correlated transcriptional profile indicate common function. Identified
327 protein complexes with at least two-thirds of their members present,
representing 14,165 confirmed protein-protein interactions. The median
expression profile correlation was strong (r = 0.61). High correlation
between expression profiles was associated with high STRING
protein-protein interaction confidence scores.</li>
<li>Perturbation of genes annotated with known functions led to similar
transcriptional responses, predicting roles for uncharacterized
genes.</li>
<li>Selected poorly annotated genes associated with ribosome biogenesis
were validated using CRISPRi, confirming their roles in the ribosome.
<img src="images/clipboard-1436037723.png" /></li>
</ul>
</div>
<div id="data-driven-definition-of-transcriptional-programs"
class="section level3">
<h3>Data-driven definition of transcriptional programs</h3>
<p><strong>Clustering Genes into Expression Programs</strong></p>
<ul>
<li><p>Purpose: To identify groups of genes that are co-regulated and
likely involved in the same biological processes or pathways.</p></li>
<li><p>Gene Co-regulation: Genes that are co-regulated tend to be
involved in the same cellular processes or pathways. By clustering these
genes together, researchers can identify which genes work together and
uncover the underlying biological mechanisms.</p></li>
<li><p>Expression Programs: This clustering helps define transcriptional
programs, which are sets of genes that are co-expressed under certain
conditions or in response to specific perturbations. Understanding these
programs provides insights into how cells respond to various stimuli and
perturbations.</p></li>
</ul>
<p><strong>Clustering Perturbations Based on Transcriptional
Profiles</strong></p>
<ul>
<li><p>Purpose: To categorize the effects of genetic perturbations based
on the changes they induce in gene expression.</p></li>
<li><p>Transcriptional Profiles: Each genetic perturbation can have a
unique impact on gene expression. By clustering perturbations with
similar transcriptional profiles, researchers can group perturbations
that induce similar changes, suggesting they affect similar pathways or
processes.</p></li>
<li><p>Functional Consequences: This clustering helps in understanding
the functional consequences of different genetic perturbations.
Perturbations that cluster together likely influence the same or related
biological pathways, making it easier to infer their roles and
effects.</p></li>
</ul>
<p><strong>Combined Benefits</strong></p>
<ul>
<li><p>Map of Biological Processes: Combining both types of clustering
allows researchers to create a comprehensive map of biological processes
and how they are regulated. Clustering genes reveals the biological
programs, while clustering perturbations shows how different genetic
changes impact these programs.</p></li>
<li><p>Identifying Key Regulators: By examining which perturbations
affect specific gene clusters, researchers can identify key regulators
and pathways involved in particular biological functions or
responses.</p></li>
<li><p>Functional Annotation: This dual approach facilitates the
functional annotation of genes and perturbations. Genes can be annotated
based on the expression programs they belong to, and perturbations can
be annotated based on the transcriptional changes they induce.</p></li>
<li><p>Predicting Gene Function: Clustering genes and perturbations
together allows for the prediction of gene function. If a perturbation
is known to affect a certain pathway, genes co-regulated in response to
this perturbation can be inferred to play roles in that
pathway.</p></li>
</ul>
<p><img src="images/clipboard-2736488339.png" /></p>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
