<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />



<meta name="date" content="2024-06-24" />

<title>Summary: Discovery of target genes and pathways at GWAS loci by pooled single-cell CRISPR screens</title>

<script src="site_libs/header-attrs-2.27/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.13.2/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-6.4.2/css/all.min.css" rel="stylesheet" />
<link href="site_libs/font-awesome-6.4.2/css/v4-shims.min.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>









<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">myproject</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/jliucx/myproject">
    <span class="fab fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Summary: Discovery of target genes and
pathways at GWAS loci by pooled single-cell CRISPR screens</h1>
<h4 class="date">2024-06-24</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2024-07-07
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 2
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>myproject/</code> <span
class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.1). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date
</a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git
repository, you know the exact version of the code that produced these
results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomjliucxmyprojecttree08dd4b6e890b7a29e61bf92a4486491e11847923targetblank08dd4b6a">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong>
<a href="https://github.com/jliucx/myproject/tree/08dd4b6e890b7a29e61bf92a4486491e11847923" target="_blank">08dd4b6</a>
</a>
</p>
</div>
<div
id="strongRepositoryversionstrongahrefhttpsgithubcomjliucxmyprojecttree08dd4b6e890b7a29e61bf92a4486491e11847923targetblank08dd4b6a"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version
<a href="https://github.com/jliucx/myproject/tree/08dd4b6e890b7a29e61bf92a4486491e11847923" target="_blank">08dd4b6</a>.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/

Untracked files:
    Untracked:  analysis/images/

Unstaged changes:
    Modified:   README.md

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were
made to the R Markdown (<code>analysis/Analysis1.Rmd</code>) and HTML
(<code>docs/Analysis1.html</code>) files. If you’ve configured a remote
Git repository (see <code>?wflow_git_remote</code>), click on the
hyperlinks in the table below to view the files as they were in that
past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/jliucx/myproject/310bb1d231955ab8106c544cad86ae977c781c40/docs/Analysis1.html" target="_blank">310bb1d</a>
</td>
<td>
jliucx
</td>
<td>
2024-06-27
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/jliucx/myproject/blob/61bf8c7b194a72232794440abef03dad6da5a7c3/analysis/Analysis1.Rmd" target="_blank">61bf8c7</a>
</td>
<td>
jliucx
</td>
<td>
2024-06-27
</td>
<td>
Complete first analysis
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/jliucx/myproject/d467b9e7838b92f5b556397c98f26281ddb6d588/docs/Analysis1.html" target="_blank">d467b9e</a>
</td>
<td>
jliucx
</td>
<td>
2024-06-27
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/jliucx/myproject/blob/f021c4a1d35cb27927aaa23c25cb7c6aa48ace1a/analysis/Analysis1.Rmd" target="_blank">f021c4a</a>
</td>
<td>
jliucx
</td>
<td>
2024-06-27
</td>
<td>
Complete first analysis
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/jliucx/myproject/5d9da0b0d97516166337364ee5e87fc7ca396686/docs/Analysis1.html" target="_blank">5d9da0b</a>
</td>
<td>
jliucx
</td>
<td>
2024-06-24
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/jliucx/myproject/3fbeeb513c3035d587e10528c56bd2c464c30376/docs/Analysis1.html" target="_blank">3fbeeb5</a>
</td>
<td>
jliucx
</td>
<td>
2024-06-24
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/jliucx/myproject/blob/5cd12976a41666fc5bafed71aec0334359775941/analysis/Analysis1.Rmd" target="_blank">5cd1297</a>
</td>
<td>
jliucx
</td>
<td>
2024-06-24
</td>
<td>
Add first summary
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<div id="part-1.-terminologies" class="section level2">
<h2>Part 1. Terminologies</h2>
<div id="massively-parallel-crispr-screen" class="section level3">
<h3><strong>Massively parallel CRISPR screen</strong></h3>
<p>A massively parallel CRISPR screen is a high-throughput method used
to investigate the function of genes on a large scale. Here’s a
breakdown of the key components:</p>
<ol style="list-style-type: decimal">
<li><p><strong>CRISPR (Clustered Regularly Interspaced Short Palindromic
Repeats)</strong>: This is a powerful technology that allows for the
precise editing of DNA. By using a guide RNA (gRNA) to target specific
DNA sequences, the CRISPR-associated protein (Cas9) can make cuts at
these locations, allowing researchers to delete, insert, or modify
genetic material.</p></li>
<li><p><strong>Screen</strong>: In the context of genetic research, a
screen refers to an experimental approach where many different genetic
modifications are tested simultaneously to observe their effects. This
is often used to identify genes that are involved in particular cellular
processes or diseases.</p></li>
<li><p><strong>Massively Parallel</strong>: This refers to the ability
to perform many (often thousands or millions) of CRISPR edits in a
single experiment. The term “massively parallel” highlights the scale
and efficiency of the process.</p></li>
</ol>
<p><strong>Notice</strong>: In a CRISPR screen, it is possible for each
cell to receive more than one guide RNA (gRNA). This approach, known as
a multiplexed CRISPR screen, allows for the simultaneous targeting of
multiple genes within the same cell.</p>
</div>
<div id="fine-mapping" class="section level3">
<h3>Fine mapping</h3>
<p>Fine-mapping is the process of identifying specific genetic variants
that are most likely to be causal within a region identified by
genome-wide association studies (GWAS). This is usually followed by
functional genomics analysis to prioritize variants that are located in
regions likely to influence gene expression or function, thereby
identifying those with higher potential to be causal.</p>
</div>
<div id="cis--and-trans-regulatory-elements" class="section level3">
<h3>Cis- and Trans-Regulatory Elements:</h3>
<ul>
<li><p>Cis-Regulatory Elements (CREs): DNA sequences located near the
genes they regulate (on the same chromosome). They often include
promoters and enhancers.</p></li>
<li><p>Trans-Regulatory Elements: Factors, such as transcription factors
or microRNAs, that regulate genes located on different chromosomes or
far from the genes they regulate.</p></li>
</ul>
</div>
</div>
<div id="part-2.-rationale-for-experiment-design"
class="section level2">
<h2>Part 2. Rationale for Experiment Design</h2>
<ul>
<li><p>Using multiple perturbations at the same locus helps to
distinguish which variants are truly causal by observing the effects of
each change. In GWAS study, not all of these variants are directly
causing the trait or disease; some are simply linked to causal variants
due to their proximity on the chromosome i.e.LD.</p></li>
<li><p>They combined systematic targeted inhibition of noncoding GWAS
loci coupled with single-cell sequencing (STING-seq), to identify target
genes at noncoding GWAS loci using single-cell pooled CRISPR
screens.</p></li>
<li><p>For a subset of validated CREs, they also inserted specific GWAS
variants using base-editing STING-seq (beeSTING-seq) and quantified the
effect size and direction of GWAS variants on gene expression. Direct
variant insertion rather than slicing the CREs can identify the
magnitude and direction of the effect on gene expression.</p></li>
<li><p>To identify candidate CREs from GWAS, the integration of
functional genomic data yielded a large set of targetable variants from
UKBB and BCX GWASs (table S1, C and D). The variants that we selected
were often the highest-probability variant in a fine-mapped GWAS locus
(294 variants) or among the 10 most probable variants (249 variants). We
also prioritized variants from non-European ancestries.</p></li>
<li><p>To disentangle loci with multiple target genes in <em>cis</em>,
they combined targeted CRE inhibition and gene inhibition. CRE
inhibition is to find out what genes are regulated by the CRE. Gene
inhibition is to find relationships among those genes, which can help to
identify whether the gene is directly regulated by the CRE or
not.</p></li>
<li><p>To explore whether the genes in the trans-regulatory networks
also play roles in blood traits and harbor cis-regulatory genetic
variants, they identified putative causal genes for several GWAS studies
by selecting protein-coding genes close to fine-mapped GWAS loci. They
grouped these genes based on their roles in different blood cell types
(platelets, RBCs, WBCs), and checked for enrichments of blood cell trait
GWAS genes in the trans-regulatory networks. Nearly all trans-regulatory
networks showed enrichment for blood cell trait GWAS genes, indicating
that these genes’ known roles in blood cell development are mediated
through their effects on regulatory networks.</p></li>
<li><p><strong>Question: why grouping these genes based on their roles
in different blood cell types (platelets, RBCs, WBCs)</strong></p>
<p><strong>Answer:</strong> Different cell types are regulated by
distinct sets of transcription factors, enhancers, and other regulatory
elements. Grouping genes by cell type allows for the identification of
cell-type-specific regulatory mechanisms. The transcription factors
GFI1B, NFE2, IKZF1, and RUNX1 might regulate different sets of genes in
platelets compared to RBCs or WBCs. Understanding these specific
regulatory mechanisms can provide insights into how these cell types
develop and function.</p></li>
<li><p><strong>Question: why multiple perturbations at same
loci?</strong></p>
<p><strong>Answer:</strong> Multiple perturbations can help identify
specific regulatory elements within a locus that are responsible for
changes in gene expression or narrow down the specific variant or small
set of variants that are truly causal.</p></li>
<li><p><strong>Question: why design 3 kinds of control
gRNA?</strong></p>
<p><strong>Answer</strong>:</p>
<p>1. Negative Controls (Nontargeting Controls)</p>
<p>Negative controls are designed to not target any gene in the genome.
These controls help to account for any non-specific effects caused by
the delivery and expression of the CRISPR components. This includes
potential off-target effects, immune responses, or general cellular
stress responses that might be induced by the presence of the CRISPR
machinery.</p>
<p>2. Positive Controls (Targeting Highly-Expressed Genes at TSSs)</p>
<p>Positive controls target known genes, often those that are highly
expressed and well-characterized. Targeting the transcription start
sites (TSSs) of these genes ensures that the CRISPR system is actively
working. These controls confirm that the CRISPR system is functional and
capable of inducing the expected effects. By targeting highly-expressed
genes, researchers can easily observe the phenotypic changes or measure
the gene expression changes, validating that the CRISPR editing is
successful.</p>
<p>3. Controls to Estimate Average Number of Perturbations (Multiple
gRNAs Targeting a Gene Encoding a Ubiquitously Expressed Cell Surface
Protein, CD55)</p>
<p>These controls are designed to estimate the efficiency and average
number of perturbations per cell. By targeting a ubiquitously expressed
gene like CD55, researchers can use flow cytometry to measure how many
gRNAs have successfully introduced changes in each cell. This helps in
understanding the multiplicity of infection (MOI) and the overall
effectiveness of the CRISPR library. Knowing the average number of
perturbations per cell is crucial for interpreting the results of the
screen accurately and ensuring that the observed effects are due to the
intended genetic modifications.</p></li>
</ul>
</div>
<div id="part-3.-workflow" class="section level2">
<h2>Part 3. Workflow</h2>
<ul>
<li>Intersecting fine-mapped GWAS variants with biochemical hallmarks of
enhancer activity to find cCREs.</li>
<li>Use STING-seq to perturb CREs and find target genes.</li>
<li>Targeting multiple CREs in some locus reveals nonfunctional linkage
disequilibrium proxies</li>
<li>Direct GWAS variant insertion with beeSTING-seq to identify the
magnitude and direction of the effect on gene expression.</li>
<li>Identify <em>trans</em>-effect networks of noncoding loci when
<em>cis</em> target genes encoded transcription factors or
microRNAs</li>
</ul>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
